### PR TITLE
ficheros

### PR DESCRIPTION
**Pull request by SandraTejedor**

### CODE REVIEW

File: Assignment2/SandraTejedor.ttl [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@prefix class: <http://assignment2.org/class#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+class:Measurement8401 property:atTime "2010-06-12T12:00:12"^^<xsd:dateTime> ;
```
Comment (ocorcho) [inline Assignment2/SandraTejedor.ttl]: It is xsd:dateTime, not <xsd:dateTime>. The same for the next line

Comment (SandraTejedor) [reply Assignment2/SandraTejedor.ttl]: I can't see the error you are referring to

Comment (ocorcho) [reply Assignment2/SandraTejedor.ttl]: Oh, sorry, in the comments the symbols "<" and ">" did not appear. I was referring to the fact that xsd:dateTime does not have to be included inside those symbols.

**Comment by ocorcho** [comment]
Changes have not been done, so I am closing the pull request.

**Pull request closed without merging**